core/src/main/java/com/scalar/db/io/BigIntColumn.java

```diff
@@ -31,6 +33,7 @@ private BigIntColumn(String name) {
 
   private BigIntColumn(String name, long value, boolean hasNullValue) {
     this.name = Objects.requireNonNull(name);
+    checkArgument(value >= MIN_VALUE && value <= MAX_VALUE, "Out of range column value for BigInt");
```

Comment:
Maybe adding the actual `value` in the error message would help users?

Reply:
Thank you for your feedback! I added the actual `value` in the error message. Please take a look again when you have time!